src/pages/Detail/DetailMain/DetailMain.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ProductImg from './ProductImg';
+import DetailDesc from './DetailDesc';
+import './DetailMain.scss';
+
+const DetailMain = ({ productBox }) => {
+  return (
+    <div className="detailMain">
+      <div className="productImg-container">
+        {productBox[0] &&
+          productBox[0].img.map(product => {
```

Comment:
```suggestion
        {productBox[0]?.img.map(product => {
```
옵셔널 체이닝 문법 사용하면 동일한 동작을 좀 더 간결하게 표현할 수 있습니다! 어떻게 동작하는건지 찾아보시고 활용해보세요!